packages/thriver-accounts/lib/client/profile.js

```diff
@@ -1,15 +1,34 @@
+/**
+ * @description
+ *  Update user profile field. Makes a call to the backend
+ *  with a property it has to update and the new value
+ * @method
+ *  @param {any} fieldName
+ *  @param {any} fieldValue
+ */
+const updateProfileField = (fieldName, fieldValue) => {
+  const profile = {};
+  profile[fieldName] = fieldValue;
+
+  Meteor.call('updateUserProfile', profile, (error) => {
+    if (error) console.log(error);
```

Comment:
Any way to notify the user when there is an error?  You could throw an `aside` with classes `error-msg hide` at the top of the template, for example, that you could populate with the error message and unhide when there was an error.  At least notifying the user the update didn't work.

Reply:
okay i will take a look into it.